### PR TITLE
Update _helpers.tpl

### PR DESCRIPTION
**Pull request by Taranasaur**
T missing from tls

### CODE REVIEW

**Comment by RafalKorepta** [comment]
Hi @Taranasaur,

Could you sign CLA and bump https://github.com/redpanda-data/helm-charts/blob/bf2ffcf6eb27801dff98b85d1ae9821e75a33c85/charts/redpanda/Chart.yaml#L26?

Thank you for contribution.

**Comment by Taranasaur** [comment]
To bump do I cancel this request and resubmit...

**Review by RafalKorepta — APPROVED** [review]

File: charts/redpanda/templates/_helpers.tpl [hunk 1]
```diff
@@ -640,7 +640,7 @@ return licenseSecretRef.key checks deprecated values entry if current values emp
   mountPath: {{ printf "/etc/tls/certs/%s" $name }}
     {{- end }}
 - name: mtls-client
-  mountPath: /etc/ls/certs/{{ template "redpanda.fullname" $ }}-client
```
Comment (alejandroEsc) [inline charts/redpanda/templates/_helpers.tpl:643]: good catch!

**Review by alejandroEsc — APPROVED** [review]
lgtm

**Pull request merged**